labs/modern-pharmacy-management.md

```diff
@@ -0,0 +1,20 @@
+# Lab Name
+Modern Pharmacy Management
+
+# Short Description
+This Lab will focus on leveraging blockchain to transform and modernize prescriptions management by enabling flexibility and managing fraud.
+
+# Scope of Lab
+The scope of the lab is to develop a proof of concept (POC) for a blockchain-based presecription management solution that covers general use case in healthcare around prescriptions and its fullfilment.
+This repo will have a end user application and the blockchain built using Hyperledger Fabric. The end user application using Hyperledger Composer will be created to showcase the use of POC created. 
+
+# Initial Committers
+- https://github.com/dewanz
+- https://github.com/kentydotcom
+- Anyone is welcome to join. 
+
+# Sponsor
+Arnaud J Le Hors rlehors@us.ibm.com
```

Comment:
I believe Arnaud's email is just lehors@...

Reply:
Tracy I have updated Arnaud's email id. Thanks for catching that.